frontend/app/src/composables/blockchain/index.ts

```diff
@@ -48,9 +48,13 @@ interface AccountAdditionFailure {
   account: AccountPayload | XpubAccountPayload;
 }
 
+interface AddAccountsOption {
+  wait: boolean;
+}
```

Comment:
when is wait used here?

Reply:
for the `addAccounts` previously it used `startPromise` but I can't track if it's actually done or not,
since I want to do it with `awaitParallelization`